cypress/integration/components/search-field.spec.js

```diff
@@ -0,0 +1,19 @@
+describe("Testa comportamento do componente SearchInput", () => { 
+    it("Retorna se o componente está visível", () => {
+        cy.visit('/')
+        cy.get('.styles_container__IveJR').should("be.visible")
+        cy.get('.styles_container__IveJR button').should("be.visible")
+        cy.get('.styles_container__IveJR input').should("be.visible")
+
+    })
+
+    it("Testa o funcionamento entrada", () => {
+        cy.get('.styles_container__IveJR input').type('alá rodrigo do lindo olhar').wait(750)
```

Comment:
se quiser adicionar um pequeno delay ao digitar cada letra (750 milissegundos):
```suggestion
cy.get('.styles_container__IveJR input').type('alá rodrigo do lindo olhar', {delay: 750});

```